Version/0.0.2

zenit70 opened the pull request:
Request for release of version 0.0.2

RiccardoM commented on src/assets/translations/it/labels.js:
```diff
@@ -2,40 +2,64 @@ export const labels = {
   description: "Descrizione",
+  details: "Dettagli",
   did: "DID",
+  fee: "Onorario",
```
Riusciamo a cambiare la traduzione in "Fee"? Non credo che "Onorario" sia la traduzione migliore

RiccardoM commented on src/assets/translations/it/labels.js:
```diff
@@ -2,40 +2,64 @@ export const labels = {
   sender: "Mittente",
+  share: "Quota",
   signer: "Firmatario",
+  status: "Status",
```
Potremmo cambiare questa traduzione in "Stato"

RiccardoM commented on src/assets/translations/it/labels.js:
```diff
@@ -2,40 +2,64 @@ export const labels = {
+  fee: "Onorario",
+  finalDate: "Data finale",
   firstUser: "Primo utente",
   fromAddress: "Da indirizzo",
```
Possiamo cambiare questa traduzione in "Mittente"?

RiccardoM commented on src/assets/translations/it/labels.js:
```diff
@@ -2,40 +2,64 @@ export const labels = {
+  status: "Status",
   submitTime: "Orario di presentazione",
+  time: "tempo",
   toAddress: "Ad indirizzo",
```
Possiamo cambiare questa traduzione in "Destinatario"?

RiccardoM commented on src/assets/translations/it/labels.js:
```diff
@@ -2,40 +2,64 @@ export const labels = {
   signer: "Firmatario",
+  status: "Status",
   submitTime: "Orario di presentazione",
+  time: "tempo",
```
Se questa e' la traduzione usata per il tempo dall'ultimo blocco, possiamo cambiarla in "Tempo dall'ultimo blocco"? Non credo sia molto chiara al momento

RiccardoM commented on src/assets/translations/it/labels.js:
```diff
@@ -2,40 +2,64 @@ export const labels = {
   availables: "Disponibili",
+  bonded: "Vincolati",
   commission: "Commissione",
   commissionRate: "Tasso commissione",
```
Possiamo cambiare questa traduzione in "Percentuale commissione"?

RiccardoM commented:
@zenit70 Can we get a short description of the most significant changes that have been made into these commits?

zenit70 commented:
> @zenit70 Can we get a short description of the most significant changes that have been made into these commits?

@RiccardoM 

- Reviewed the management and sequence of calls to the Cosmos API.
- Updated the style of all sections to that of the dashboard.
- Fixed validator and account sections to display correct data.
- Implemented pagination for block, validator and account tables.
- Added tests for 'components'.
